### ISSUE TITLE
Supports are not tracked in the database

### ISSUE THREAD

**Issue by jackrobison** [lbryio/lbryum-server#49]
This means that lbrycrdd must be queried in order to return the supports for a claim. The same is true for the effective amount, since this depends on the current valid supports. This substantially slows `getclaimbyid` and `getclaimsbyids` (as well as other commands that call `BlockchainProcessorBase.get_claim_info`).

**Comment by shyba** [comment]
lbryumx gets supports by asking lbrycrdd, like lbryum-server, except that it uses an implementation which includes this information on the other RPC calls (https://github.com/lbryio/lbrycrd/issues/124). The speed is totally different in there and the calls to lbrycrd are unavoidable (by design, as we are solving things on lbrycrd instead).

For the effective amount from lbrycrd, a bug was fixed and tests added: https://github.com/lbryio/lbrycrd/pull/123

In terms of speed, here we added a caching layer which improved that. On lbryumx the response time so far is equivalent to a hot cache response from lbryum-server or faster. So, I think we can close this. There are points to improve in terms of speed, but what I'm seeing in lbryumx so far are issues in the size and complexity of the resolve implementation, which I'm trying to refactor.

I think this can be closed as lbryum-server won't gain supports tracking and lbryumx disabled it by design. We may reopen the discussion in lbrycrd or lbryumx if support indexing becomes a slow point.
